lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
```

Comment:
It's a bit repetitious to be checking the OS in each of these methods. Can you create a method like:

```
def cmd_for_os
  if inspec.os.redhat? || inspec.os.name == 'fedora'
    '/sbin/modprobe'
  else
    'modprobe'
  end
end
```

And then use that in each of these methods?

```
modprobe_cmd = "#{cmd_for_os} --showconfig | grep blacklist | grep #{@module}"
```